thirdeye/thirdeye-frontend/app/pods/components/rootcause-metrics/component.js

```diff
@@ -55,27 +50,27 @@ export default Ember.Component.extend({
       const metricUrns = filterPrefix(Object.keys(entities), ['thirdeye:metric:']);
       let output = [];
 
-      if (sortProperty == ROOTCAUSE_METRICS_SORT_PROPERTY_METRIC) {
-        output = metricUrns.map(urn => [metrics[urn], urn]).sort().map(t => t[1]);
+      if (sortProperty === ROOTCAUSE_METRICS_SORT_PROPERTY_METRIC) {
```

Comment:
I feel that a `switch case` might be cleaner here

Reply:
fixed